### PR TITLE
separate out brun to a separate python file

### PR DESCRIPTION
**Pull request by arvidn**
as a much simple wrapper around `run_chia_program()`.
No more shenanigans with cost offsets or launcher programs. This makes `-c` report accurate and expected costs, which is the main rationale for doing this. `run` is preserved to behave as it does for now. To get to the previous `brun` behavior, you can still run `run -s0`, which I made some of the tests use now.

### CODE REVIEW

File: tests/cmd/numerical.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-brun -n '(q (1 2 3))'
+run -s0 -n '(q (1 2 3))'
```
Comment (richardkiss) [inline tests/cmd/numerical.txt:1]: Why not just add `-n` to the new `brun`? It's frequently useful.

File: tests/stage_2/add-1.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-brun -s2 '(opt (com (q . (+ 6 55))))'
+run -s0 -s2 '(opt (com (q . (+ 6 55))))'
```
Comment (richardkiss) [inline tests/stage_2/add-1.txt:1]: This seems like a victim of automated search and replace.
```suggestion
run '(opt (com (q . (+ 6 55))))'
```

**Comment by richardkiss** [comment]
I would really miss `brun -v` and `brun -y main.sym` for debugging, and I use them a lot, and suspect other developers do too. Given the choice, I'd prefer they continue to be in `brun` and removed from `run`.

**Comment by richardkiss** [comment]
If your goal is to just have `brun` return exact costs, I'd be happy to help with this. I believe it can be done surgically, reducing the amount of code rather than increasing it.

File: tests/stage_2/add-1.txt (hunk 2 above)
Comment (arvidn) [reply tests/stage_2/add-1.txt:1]: yes..

**Comment by arvidn** [comment]
I don't think `-v` or `-y` works with the rust back-end today anyway. but there's a lot of indirect logic to get to some of those

**Comment by richardkiss** [comment]
How about for now we create a new tool `clvm` or `rclvm` or `rcl` or `your idea here` that only uses the rust back end and has limited features but calculates cost exactly?

**Comment by arvidn** [comment]
yeah, I agree that it's questionable to strip down `brun` like this.
What I think is important is that the documentation and the actual tools people use are aligned. so, maybe a better approach is to tweak `launch_tool` to run the program in a similar way as this patch.

The parts that makes me unsure on how to do this are the "stages" and how the launcher program depends on it. Whereas I would like to not use a "launcher" at all. and then finally we can remove the cost_offset.

**Pull request closed without merging**